srcs/Client.cpp

```diff
@@ -61,15 +61,48 @@ void Client::recvStartLine(const std::string &line)
  */
 void Client::recvHeader(const std::string &line)
 {
-	// TODO: \n 단위로 파싱을 해서 줄별로 넘겨받습니다.
-	//       각 라인을 파싱해서 map에 넣어주시면 됩니다.
-	std::cout << "HEADER : " << line << std::endl;
+	// std::cout << "HEADER : " << line << std::endl;
+	std::vector<std::string> headers = ft_split(line, ':');
+	this->_request.insertToHeader(headers[0], line.substr(headers[0].size() + 1));
+}
+
+/**
+ * Client::setConfig
+ * @brief  RECV_HEADER가 끝난 이후, Host와 Port를 기반으로 올바른 Config 인스턴스 연결
+ * @param  {const ConfigGroup} group : Config 벡터를 가져올 ConfigGroup 인스턴스
+ */
+void Client::setConfig(ConfigGroup &group)
+{
+	std::string host = this->_request.getHeaderValue("Host");
+	if (host.size() == 0)
+		return;
+	host.erase(host.find(':'));
+	std::string path = this->_request.getStartLine().path;
+	int length = group.getServerCnt();
+	for (int i = 0; i < length; i++)
+	{
+		std::vector<Config> server_config = group.getConfig(i);
```

Comment:
레퍼런스로 받는 편이 효율이 좋을 것 같아요.

```suggestion
		std::vector<Config> &server_config = group.getConfig(i);
```